ci/travis/install-dependencies.sh

```diff
@@ -64,7 +64,7 @@ elif [[ "$PYTHON" == "3.6" ]] && [[ "$platform" == "macosx" ]]; then
     opencv-python-headless pyyaml pandas==0.24.2 requests \
     feather-format lxml openpyxl xlrd py-spy pytest pytest-timeout networkx tabulate aiohttp \
     uvicorn dataclasses pygments werkzeug kubernetes flask grpcio pytest-sugar pytest-rerunfailures pytest-asyncio \
-    blist scikit-learn numba
+    blist scikit-learn numba boto3
```

Comment:
why do we need this?

Reply:
I see, it's for boto tests later. These tests don't actually require creds right?

Reply:
Correct - the tests are stubbed/mocked out and don't interact with any real AWS account.